src/Symfony/Bridge/Doctrine/Attribute/MapEntity.php

```diff
@@ -47,6 +47,7 @@ public function __construct(
         public ?string $message = null,
     ) {
         parent::__construct($resolver, $disabled);
+        $this->withDefaults($this, null); // to trigger the validation logic
```

Comment:
Wouldn't it be nicer to extract the validation into a dedicated method, that can be called both here and in the `withDefaults` function?

Reply:
updated